src/app/pages/RewardPage/index.tsx

```diff
@@ -32,25 +62,58 @@ export function RewardPage() {
 
       <Header />
 
-      <div className="tw-container tw-mt-9 tw-mx-auto tw-px-6">
+      <StyledBackgroundImageWrapper>
+        <img src={imgSov} alt="SOV" />
+        <img src={imgBtc} alt="BTC" />
+      </StyledBackgroundImageWrapper>
+      <div className="tw-container tw-mt-9 tw-mx-auto tw-px-6 tw-relative">
         <div className="tw-mt-4 tw-items-center tw-flex tw-flex-col">
-          <div className="tw-w-full tw-items-center tw-justify-center tw-flex tw-flex-col tw-space-y-8 lg:tw-flex-row lg:tw-space-x-8 lg:tw-space-y-0">
-            <ClaimForm address={userAddress} />
-            <StakingRewardsClaimForm address={userAddress} />
+          <div className="tw-w-230">
+            <div className="tw-flex tw-flex-row tw-items-center tw-justify-start tw-mt-24">
+              <div className="tw-w-full">
+                <Tab
+                  text={t(translations.rewardPage.sov.reward)}
+                  amount={rewardSov}
+                  active={activeTab === RewardTabType.REWARD_SOV}
+                  onClick={() => setActiveTab(RewardTabType.REWARD_SOV)}
+                />
+              </div>
+              <div className="tw-w-full">
+                <Tab
+                  text={t(translations.rewardPage.sov.liquid)}
+                  active={activeTab === RewardTabType.LIQUID_SOV}
+                  onClick={() => setActiveTab(RewardTabType.LIQUID_SOV)}
+                  amount={`${weiToNumberFormat(liquidSovClaimAmount, 6)} SOV`}
+                />
+              </div>
+              <div className="tw-w-full">
+                <Tab
+                  text={t(translations.rewardPage.sov.fee)}
+                  isDisabled
+                  amount="Coming soon"
```

Comment:
Please add this as a string so I can translate to PT-BR. Thanks

Reply:
Don't you worry, this is just a work in progress, I'll use translation strings before it's final.